content/22.vaccines.md

```diff
@@ -263,6 +263,68 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
+A select few candidates were further investigated as a single dose regimen in Syrian golden hamsters, a single dose regimen in rhesus macaques, and a single- and two-dose regimen in both adult and aged rhesus macaques [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258].
+From these studies, the JNJ-78436735 candidate was selected for its favourable immunogenicity profile and for its ease of manufacturability [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258].
+A SARS-CoV-2 challenge study in rhesus macaques showed that vaccine doses as low as 2 x 10^9^ viral particles/mL was sufficient to induce strong protection in bronchoalveolar lavage but that higher doses of 1.125 x 10^10^ were required to close achieve close to complete protection in nasal swabs  [@doi:10.1101/2021.01.27.428380]. 
+Indeed, six months post-immunization, levels of S-binding and neutralizing antibodies in rhesus macaques indicated that the JNJ-78436735 vaccine conferred durable protection against SARS-CoV-2 [@doi:10.1101/2021.01.30.428921].
+
+Following selection of the JNJ-78436735 vaccine, J&J began phase I/IIa trials. 
+The interim phase I/IIa data was placed on the _medRxiv_  preprint server on September 25th, 2020 [@url:10.1101/2020.09.23.20199604], which was later published in the _New England Journal of Medicine_ on January 13th, 2021 [@doi:10.1056/NEJMoa2034201].
+The phase I/IIa multi-center, randomized, placebo-controlled trial enrolled 402 healthy participants between 18-55 years old and a further 403 healthy older participants ≥ 65 years old [@doi:10.1056/NEJMoa2034201].
+A dose of either 5 x 10^10^ (low dose) or 1 X 10^11^ (high dose) viral particles per mL was administered intramuscularly as part of either a single dose or two-dose regimen 56 days apart. 
+Those who received only one dose of either vaccine received a placebo dose at their second vaccination visit. 
+A comparison of the single versus the double dose regimen has yet to be published. 
+The primary endpoints of both the trial were safety and reactogenicity of each dose. 
+Fatigue, headache, myalgia, and pain at the injection site were the most frequent solicited adverse events reported by participants. 
+Although less common, particularly for those in the elderly cohort and those on the low dose regimen, the most frequent systemic adverse effect was fever. 
+Overall, it seems that immunization was well tolerated, particularly at the lower dose concentration.
+
+In terms of reactogenicity, over 90% of those who received either the low or high dose demonstrated seroconversion in a neutralization assay using wild-type SARS-CoV-2, 29 days after immunization. 
```

Comment:
I put it in as it is one of the main endpoints for reactogenicity... but we can use more sweeping statements generally about the endpoints to reduce content if necessary

Reply:
It's probably okay to leave it in at this stage.  It can be easier to have a comprehensive draft merged in and then assess whether we want to be more concise anywhere.